tests/utils/test_setup.c

```diff
@@ -38,6 +38,23 @@ SF_CONNECT *setup_snowflake_connection() {
             "UTC", SF_BOOLEAN_TRUE);
 }
 
+/**
+ * Enables C_API_QUERY_RESULT_FORMAT='ARROW_FORCE'.
+ * This can be removed when Arrow support is officially added.
+ */
+SF_STATUS STDCALL enable_arrow_force(SF_CONNECT *sf) {
```

Comment:
Here you define it with STDCALL and in the header file you do not. In Linux, STDCALL is a no-op, but its important in Windows. Try adding STDCALL to the header file too

Reply:
Ah, I see that. Thanks!